scripts/fork-and-cast-spell.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -e
+
+trap "kill 0" EXIT
+
+spell_path_or_address=$1
+# default params for gnosis 
+spell_executor=${2-0xc4218C1127cB24a0D6c1e7D25dc34e10f2625f5A}
```

Comment:
lets get rid of these default parameters. Can we raise an error instead if they are missing?